# Example parameters for a UART connection to the sic_mpu9250 compute module.
sic_mpu9250:
  ros__parameters:
    # serial_port: "/dev/serial/by-path/pci-0000:00:14.0-usb-0:1.3:1.0-port0"
    serial_port: "/dev/ttyUSB0"
    frame_id: "imu"
    publish_frequency: 10.0
    publish_tf_on_map_frame: false